prow/jobs/jinja/presubmits/code_gen_tests.jinja2

```diff
@@ -0,0 +1,58 @@
+  aws-controllers-k8s/code-generator:
+  - name: unit-test
+    decorate: true
+    optional: false
+    always_run: true
+    spec:
+      serviceAccountName: pre-submit-service-account
+      containers:
+      - image: public.ecr.aws/m5q3e4b2/prow:prow-unit-0.0.1
+        resources:
+          limits:
+            cpu: 1
+        command: ["make", "test"]
+
+{% for service in code_gen_presubmit_services  %}
+  - name: {{ service }}-controller-test
```

Comment:
One test for every single service is going to greatly increase the number of checks on every PR, and greatly increase the load on Prow. I'm fine with this for now but we may want to rethink how this scales, or how we can parallelise these loads, in the future.

Reply:
Definitely. It does not make sense to run for all AWS services. But we will pick our selected controllers in future.